outport/process/outportDataProvider.go

```diff
@@ -125,6 +130,13 @@ func (odp *outportDataProvider) PrepareOutportSaveBlockData(arg ArgPrepareOutpor
 }
 
 func collectExecutedTxHashes(bodyHandler data.BodyHandler, headerHandler data.HeaderHandler) (map[string]struct{}, error) {
+	if check.IfNil(bodyHandler) {
```

Comment:
maybe these checks can be removed because the body and header are already checked on lines `76` and `79`